jsoc/gsoc/tables.md

```diff
@@ -48,4 +48,37 @@
 
 **Expected Results:** Depends on the specific projects we would agree on.
 
-**Mentors:** [Tanmay Mohapatra](https://github.com/tanmaykm)
\ No newline at end of file
+**Mentors:** [Tanmay Mohapatra](https://github.com/tanmaykm)
+
+## DataFrames.jl join enhancements
+
+**Difficulty**: Hard
+
+**Duration**: 175 hours 
+
+[DataFrames.jl](https://github.com/JuliaData/DataFrames.jl) is one of the more popular implementations of tabular data type for Julia. One of the features it supports is data frame joining. However, more work is needed to improve this functionality. The specific targets for this project are (a final list of targets included in the scope of the project can be decided later).
```

Comment:
Hey @bkamins, ideally we would generally scope out these projects, the Google team advises against just including a list of open issues unless they are specific for GSoC project options, can you port over a high level work flow on a few of these?